### ISSUE TITLE
Back port to Win-10 vanilla (10.0.10586)

### ISSUE THREAD

**Issue by anormann** [sdias/win-10-virtual-desktop-enhancer#45]
At work I am not able to update Win-10, but I use this at home and it's just perfect.

Is it at all possible to not require anniversary, and if so could you hint at what I could do to make this work? I'm guessing it's to do with the .dll but can't see that there is source included. I really only care about moving a window to another desktop + switch to that.

**Comment by sdias** [comment]
Hi. Thanks.
I think that functionality has been there since the very beginning, so you might want to try one of the older releases. I should have a disclaimer saying which version requires the Anniversary update, you can try the one just before that.
If that doesn't work for you, you can try using the latest release with one of the older DLLs; some of the functionality won't work, but most will.

**Comment by anormann** [comment]
Thanks for the tip.

Tried using the DLL from 0.6.2 with the latest. Tooltips work, as well as (obviously) desktop switching and wrap. But I can't get `MoveWindowAndSwitchToDesktop` to work. It seems MoveAndSwitch was introduced in 0.7.0 which needs Anniversary.

**Comment by sdias** [comment]
Yeah... I just rechecked and unfortunately that feature wasn't available in the version of the underlying library that this project used for vanilla Windows 10. Sorry for leading you on, and thanks for reporting back.

**Comment by sdias** [comment]
Nevermind my previous comment, it didn't make any sense. `MoveWindowAndSwitchToDesktop` was introduced in that version, but in the underlying code I'm just doing the two different operations (move a window and switch to a desktop) at once, with two separate API calls, so there is no reason why it shouldn't work. I'll have to setup an environment to test this properly, but I'll look into this at some point.
